server/game/GameActions/DiscardPower.js

```diff
@@ -6,7 +6,7 @@ class DiscardPower extends GameAction {
     }
 
     canChangeGameState({ card }) {
-        return ['faction', 'play area'].includes(card.location) && card.power > 0;
+        return ['active plot', 'faction', 'play area'].includes(card.location) && card.power > 0;
```

Comment:
Feels like this array is duplicated a lot and could be a return from a function so you wouldn't need to change it in so many places, maybe one for the future